examples/message_passing/after.cpp

```diff
@@ -0,0 +1,59 @@
+// This example shows how to use caf::after
+
+#include "caf/all.hpp"
+
+#include <algorithm>
+#include <chrono>
+#include <iostream>
+#include <random>
+using std::cout;
+using std::endl;
+
+using namespace caf;
+
+// Sends random characters to buddy, and then waits for a letter back
+behavior noisy_buddy(event_based_actor* self, actor buddy) {
+  using namespace std::chrono_literals;
+  std::random_device rd;
+  std::minstd_rand gen{rd()};
+  const auto count = std::uniform_int_distribution<>(20, 100)(gen);
```

Comment:
Please stick to uniform initialization syntax.

```suggestion
  const auto count = std::uniform_int_distribution<>{20, 100}(gen);
```